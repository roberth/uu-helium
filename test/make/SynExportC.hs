type P = Int -> Int

data A = A deriving Show
