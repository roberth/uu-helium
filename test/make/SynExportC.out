Prelude is up to date
Compiling make/SynExportC.hs
Compilation successful
No 'main' function defined in this module
